docs/dotnet-configuration.md

```diff
@@ -4,6 +4,91 @@ title: Scope .NET Agent configuration
 sidebar_label: Configuration
 ---
 
+The behaviour of the Scope Agent can be modified by using  environment variables or by using a configuration file `Scope.yml`in the source root of the project.
+
+> These configuration settings can also be provided via CLI flags. Run `scope-run --help` for more information.
+
+
+
+## Disabling Scope instrumentation
+
+You can disable Scope auto instrumentation. Scope auto instruments your code and tests by default, you can disable this behaviour.
+
+### Using Environment Variables
+
+```sh
+SCOPE_AUTO_INSTRUMENT=False
+```
+
+### Using YAML Configuration file
+
+```yaml
+scope:
+  auto_instrument: false
```

Comment:
wasn't this 
```yaml
scope:
  instrumentation:
    enabled: true
```
or is this different?

Reply:
I will check, I copied it from some other platform

Reply:
In Java is:
```
scope:
  instrumentation:
    enabled: true
```

Reply:
Changed to:
 scope:
  instrumentation:
    enabled: false